nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/GetHDFSFileInfo.java

```diff
@@ -183,9 +186,8 @@
             .name("gethdfsfileinfo-batch-size")
             .description("Number of records to put into an output flowfile when 'Destination' is set to 'Content'"
                 + " and 'Group Results' is set to 'None'")
-            .required(true)
-            .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
-            .defaultValue("1")
+            .required(false)
+            .addValidator(StandardValidators.createAllowEmptyValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR))
```

Comment:
POSITIVE_INTEGER_VALIDATOR seems to me more reasonable here.
"0" falls back to batch size = 1, but it might not be straightforward / obvious for the end user.
The batch size properties of the other processors also have 1 as their lower bound.